runtime/vm/JFRChunkWriter.cpp

```diff
@@ -98,21 +101,32 @@ VM_JFRChunkWriter::writeJFRMetadata()
 
 	U_8 *blobStart = _bufferWriter->getCursor();
 	if (_debug) {
-		j9tty_printf(PORTLIB, "Metadata blob start offset = 0x%lX, size = %d\n", _bufferWriter->getFileOffsetFromStart(_bufferWriter->getCursor()), (int) _vm->jfrState.metaDataBlobFileSize);
+		j9tty_printf(
+				PORTLIB,
+				"Metadata blob start offset = 0x%llX, size = %zu\n",
+				_bufferWriter->getFileOffsetFromStart(_bufferWriter->getCursor()),
+				_vm->jfrState.metaDataBlobFileSize);
 	}
 
 	/* metadata blob file */
 	_bufferWriter->writeData(_vm->jfrState.metaDataBlobFile, _vm->jfrState.metaDataBlobFileSize);
 
 	if (_debug) {
-		j9tty_printf(PORTLIB, "Metadata blob size from LEB128 = %u\n",  VM_BufferWriter::convertFromLEB128ToU32(blobStart));
+		j9tty_printf(
+				PORTLIB,
+				"Metadata blob size from LEB128 = %u\n",
+				VM_BufferWriter::convertFromLEB128ToU32(blobStart));
```

Comment:
This should use `convertFromLEB128ToU64()` to allow for sizes possibly larger than 28 bits now that writeLEB128PaddedU72()` is used to write size fields.

Same comment for line 129.